apisix/script.lua

```diff
@@ -25,18 +25,23 @@ local _M = {}
 function _M.load(route, api_ctx)
     local script = route.value.script
     if script == nil or script == "" then
+        core.log.error("fail to load empty script")
         return nil
     end
 
     local loadfun = loadstring(script, "route#" .. route.value.id)
```

Comment:
we need to use a `lru` way to cache the loaded object, avoid to call `loadstring` for every request.

Reply:
anyway, we can fix this in a new PR.

Reply:
I created a new issue, we can fix this later. https://github.com/apache/apisix/issues/1995